src/probes/telem_record_gen.c

```diff
@@ -187,6 +193,17 @@ int validate_opts(void)
                 return ret;
         }
 
+        /* Event id  */
+        if(opt_event_id) {
+                if (strlen(opt_event_id) != 32) {
+                        fprintf(stderr, "Error: event_id should have 32 characters\n");
```

Comment:
You could report how many it had and what they were to avoid the user having to do so manually.

You could mention the need for the characters to be ascii, in this increasingly utf-8 world.